index.html

```diff
@@ -0,0 +1,383 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>KMarketer</title>
+    <link rel="stylesheet" href="./index.css">
+</head>
+<body>
+    <!-- Header -->
+    <header class="header">
+        <div class="header__container">
+            <h1 class="logo">KMarketer</h1>
+            <div class="header__icons">
+                <img src="./assets/search-icon.png" alt="Search Icon" class="header__search-icon">
+                <img src="./assets/cart-icon.png" alt="Cart Icon" class="header__cart-icon">
+                <img src="./assets/hamburger-icon.png" alt="Hamburger Icon" class="header__hamburger-icon"> 
+            </div>
+        </div>
+        <ul class="links">
+            <li class="links__item">Home</li>
+            <li class="links__item">Product</li>
+            <li class="links__item">Pricing</li>
+            <li class="links__item">Contact</li>
+        </ul>
+    </header>
+    <!-- Header - End -->
+
+    <!-- Main -->
+    <section class="mission">
+        <div class="note">
+            <h1 class="note__title">
+                Experts are <br>
+                here to solve <br>
+                your business <br>
+                problem.
+            </h1>
+            <p class="note__description">
+                We know how large 
+                objects will act, but things 
+                on a small scale just do 
+                not act that way.
+                Get Quote Now
+                Learn More
+            </p>
+        </div>
+        <div class="mission__btns">
+            <button class="btn btn--space-below btn--rounded-full">Get Quote Now</button>
+            <button class="btn btn--fill-none btn--rounded-full">Learn More</button>
+        </div>
+    </section>
+    <section class="choice">
+        <div class="note">
+            <h1 class="note__title note__title--uppercase note__title--text-big">
+                Why <br>
```

Comment:
<br> is a bad idea because in responsiveness you many new to display this as one line  , maybe use the width or the padding of the element instead

Reply:
I fixed this.